src/test/java/com/client/core/base/tools/entitychanger/impl/StandardEntityChangerTest.java

```diff
@@ -0,0 +1,95 @@
+package com.client.core.base.tools.entitychanger.impl;
+
+import com.bullhornsdk.data.model.entity.core.standard.Candidate;
+import com.bullhornsdk.data.model.entity.core.standard.CorporateUser;
+import com.bullhornsdk.data.model.entity.embedded.Address;
+import com.client.core.base.tools.entitychanger.EntityChanger;
+import com.google.common.collect.ImmutableMap;
+import org.junit.Test;
+
+import java.math.BigDecimal;
+import java.util.Map;
+
+import static org.junit.Assert.*;
+
+public class StandardEntityChangerTest {
+
+
+    @Test
+    public void retrieveField() {
+        EntityChanger entityChanger = new StandardEntityChanger();
+        // set up
+        Candidate candidate = new Candidate(1);
```

Comment:
You can use the `@Before` annotation to set up a single entity that you can use throughout many of your tests.

`@Before` will tell JUnit that this function will run before every single unit test, so you could have...

`
@Before
setupTests() {

    this.candidate = new Candidate(1);
    this.candidate.setFirstName("Test");
}`